lib/samsara/model.rb

```diff
@@ -12,7 +12,7 @@ def self.is_audited
       has_many :auditing_revisions, class_name: Samsara.revision_class_name, as: :subject
 
       after_create  :audit_create
-      after_update  :audit_update,  if: :changed?
+      after_update  :audit_update,  if: :saved_changes?
```

Comment:
Do you want this to fail your update if saving the audit fails? Because I don't think it does right now, `saved_changes?` returns a boolean, and in the 5.2 defaults callbacks that return a falsy value no longer halt the callback chain.

Either, change `save` to `save!`, or explicity wrap this callback so that it does `throw(:abort)` when saving an audit failed.

Reply:
actually, i'm not sure if you'd want your audits to be able to crash the system - the alternative audits we're making fail silently too afaik

Reply:
Currently in Grid, as all those callbacks go through `create_audit`, which calls the boolean returning `save`, and therefore the callback _will_ halt when saving an audit fails.

When we remove `ActiveSupport.halt_callback_chains_on_return_false = true`, which is required for the Rails 5.2 upgrade, this behaviour will change. If you say that the previous behaviour was unwanted, great, but document and write a test for that. If you say the current behaviour is wanted, add a throw and maybe also write the same test ^^.

Reply:
i realize that it's not how it currently works, which is why i'm posing the question if that's still desired behaviour - considering the alternative audits we're creating i though perhaps not. Didn't write a test since it didn't seem logical to test default rails behaviour.